solcjs-get

```diff
@@ -4,10 +4,39 @@
 // as part of the prepublish step.
 
 var pkg = require('./package.json');
-var fs = require('fs');
+var fs = require('fs-extra');
 var https = require('follow-redirects').https;
 var MemoryStream = require('memorystream');
-var keccak256 = require('js-sha3').keccak256;
+var path = require('path');
+var compilerDir = 'bin';
+
+var yargs = require('yargs')
+  .usage('Usage: [options| version]')
+  .option('list', {
+    describe: 'List all the versions',
+    type: 'bool'
+  })
+  .option('all', {
+    describe: 'Download all the versions. Additionnaly, use --nightly to get the nightly builds.',
+    type: 'bool'
+  })
+  .option('nightly', {
+    describe: 'Combined with --all, it adds the nightly to the list',
+    type: 'bool'
+  })
+  .option('releases', {
+    describe: 'Get all the release builds',
+    type: 'bool'
+  })
+  .option('clean', {
+    describe: 'Delete all the compiler in ./bin',
+    type: 'bool'
+  })
```

Comment:
These options all seem to be mutually exclusive but the script does not print any error if you use more than one. This was pretty confusing when I tried running it for the first time. Using more than one should not be allowed unless it actually does something.

BTW, I think that combining `--list` with `--all`/`--nightly`/`--releases` might actually be useful. I'd even make `--list` print only releases by default because the whole list is pretty long.